api/api/spec/spec.yaml

```diff
@@ -1052,128 +1227,139 @@ components:
           type: object
           properties:
             cores:
+              description: Number of cores the CPU has.
```

Comment:
keep type the first key in the definition

Reply:
Done in 266632160480aff4ab91873668d658ef85dcec2b